src/modules/Error404/components/Error404/components/styles/styles.scss

```diff
@@ -0,0 +1,3 @@
+.error-content {
```

Comment:
The stylesheet file should be located on the same level as the component whose styles are discribed in it.